frontend/server/src/Controllers/Contest.php

```diff
@@ -563,17 +576,14 @@ public static function getContestListMineForSmarty(
         // suggesting to contribute to the community by releasing the material to
         // the public. This flag ensures that this alert is shown only once per
         // session, the first time the user visits the "My contests" page.
-        $privateContestsAlert = false;
-        {
-            $scopedSession = \OmegaUp\Controllers\Session::getSessionManagerInstance()->sessionStart();
-            $privateContestsAlert = (
-                !isset($_SESSION['private_contests_alert']) &&
-                \OmegaUp\DAO\Contests::getPrivateContestsCount($r->user) > 0
-            );
+        $scopedSession = \OmegaUp\Controllers\Session::getSessionManagerInstance()->sessionStart();
+        $privateContestsAlert = (
+            !isset($_SESSION['private_contests_alert']) &&
+            \OmegaUp\DAO\Contests::getPrivateContestsCount($r->user) > 0
+        );
         if ($privateContestsAlert) {
             $_SESSION['private_contests_alert'] = true;
         }
```

Comment:
nit:

```suggestion
        }
        unset($scopedSession);
```

la intención del bloque era delimitar que la variable `$scopedSession` no viva después de esa línea. pero resulta que PHP no destruye las cosas cuando se salen de scope (excepto cuando regresas de una función), así que hacer el `unset()` explícito es más correcto.

Reply:
Listo